openadapt/db/crud.py

```diff
@@ -271,6 +271,11 @@ def delete_recording(session: SaSession, recording: Recording) -> None:
         recording (Recording): The recording object.
     """
     recording_timestamp = recording.timestamp
+
+    # Delete associated ActionEvent and Screenshot entries
+    session.query(ActionEvent).filter(ActionEvent.recording_id == recording.id).delete()
+    session.query(Screenshot).filter(Screenshot.recording_id == recording.id).delete()
+
```

Comment:
What do you think about this intead:

```
class Recording(Base):
    __tablename__ = 'recording'
    id = Column(Integer, primary_key=True)
    timestamp = Column(DateTime)
    action_events = relationship("ActionEvent", cascade="all, delete-orphan", backref="recording")
    screenshots = relationship("Screenshot", cascade="all, delete-orphan", backref="recording")
```

Reply:
Done here 09314d7bf478855d06bc265c3c0c9459b8591331